onboarding/computer-setup.md

```diff
@@ -154,3 +154,35 @@ export PROJECT_HOME=$HOME/
 source /usr/local/bin/virtualenvwrapper.sh #centos
 source /usr/share/virtualenvwrapper/virtualenvwrapper.sh #ubuntu
 ```
+
+## Email
+
+You can access your PolyMTL email account (@polymtl.ca) at https://zimbra.polymtl.ca, or you can add it to the macOS Mail app.
+
+## Add PolyMTL email to the macOS Mail app
```

Comment:
```suggestion
### Add PolyMTL email to the macOS Mail app
```

Reply:
Fix in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/78/commits/bd4dac25d54b19636136f1fbe843c45155c72369